src/app/components/LinkToExplorer/index.tsx

```diff
@@ -50,7 +51,7 @@ export function LinkToExplorer(props: Props) {
   return (
     <a
       className={props.className}
-      href={`${url}/tx/${props.txHash}`}
+      href={`${url}/${props.isAddress ? 'address' : 'tx'}/${props.txHash}`}
```

Comment:
I think we should have another component for addresses, like could be named `AddressBadge`